sdk/textanalytics/azure-ai-textanalytics/tests/test_auth.py

```diff
@@ -15,6 +15,7 @@ class TestAuth(TextAnalyticsTest):
     def test_active_directory_auth(self):
         token = self.generate_oauth_token()
         endpoint = self.get_oauth_endpoint()
+        raise ValueError(endpoint)
```

Comment:
seems weird. why are there more lines in this method after you raise a value error?

Reply:
thanks for catching, that was for debugging purposes and am removing it